### DISCUSSION TITLE
Does RingMQTT Support Z-Wave Door Locks that are integrated with Ring Security?

### DISCUSSION THREAD

**Discussion by supersebbo** [tsightler/ring-mqtt#775] [Support Q&A]
I have a Yale Door Lock that is integrated with Ring as a '3rd Party' device via the Alarm hub (Z-Wave device).  The lock appears as a device in the Ring App and I'm able to unlock and lock the door using the Ring App but so far it doesn't appear the Lock device is being recognised by RingMQTT.  

Does anyone have experience with this or know if it should be appearing?

**Comment by tsightler** [comment]
Z-wave locks are supported, that's why they are listed in the documentation as supported devices.  However, 99% of people that have asked about this over the years don't actually have a Z-wave lock, but rather some other Yale smartlock.  What specific model lock do you have?

**Reply by supersebbo** [reply]
So it’s a Yale smart lock with the Yale Z-Wave module installed.  https://yalehome.co.uk/z-wave-module-2/

I was a little premature in my post as I have now since restarted the add-on and the discovery kicked in and it’s showing as an MQTT device in HA.

Thanks as always for your support and dedication to this amazing piece of software!

**Comment by supersebbo** [comment]
The lock is supported, needed a restart to get discovered.
